### DISCUSSION TITLE
Home Mini Status

### DISCUSSION THREAD

**Discussion by michaeljaddy** [BottlecapDave/HomeAssistant-OctopusEnergy#682] [Ideas]
Hi there,

Is there a way to get the status of the home mini via the API? Maybe the LED light status whether it is white or green etc?

For context we have had a power cut and i have switched my home over to run off the EPS output of my inverter, i would however like Home Assistant to send me a notification when it thinks power has been restored. Currently my home mini has a green LED as it cannot connect to my smart meter, so i was hoping to use this as a trigger to say when power is restored - i.e. when it gets connectivity back to the smart meter.

Thanks in advance

Mike

**Comment by BottlecapDave** [comment]
Hello. Unfortunately there's nothing in the API that indicates this. I would suggest having an automation that monitors the `data_last_retrieved` attribute on the current demand sensor, and if the gap between now and then exceeds a defined threshold, raise a notification.

**Reply by michaeljaddy** [reply]
Perfect I will try that, many thanks!
